cmd/helm/repo_add.go

```diff
@@ -114,6 +116,16 @@ func (o *repoAddOptions) run(out io.Writer) error {
 		return errors.Errorf("repository name (%s) already exists, please specify a different name", o.name)
 	}
 
+	if o.username != "" && o.password == "" {
+		fmt.Fprint(out, "Password:")
```

Comment:
can you add a whitespace character after the colon character for formatting?

Reply:
Sure thing, I just pushed a new commit with the space.